PostFreeSurfer/scripts/CreateMyelinMaps.sh

```diff
@@ -114,6 +116,7 @@ SurfaceSmoothingFWHM="4"
 MyelinMappingSigma=`echo "$MyelinMappingFWHM / ( 2 * ( sqrt ( 2 * l ( 2 ) ) ) )" | bc -l`
 SurfaceSmoothingSigma=`echo "$SurfaceSmoothingFWHM / ( 2 * ( sqrt ( 2 * l ( 2 ) ) ) )" | bc -l`
 
+IFS='@' read -a LowResMeshesArray <<< "${LowResMeshes}"
```

Comment:
I would use `IFS=' @'` to accept spaces also (which the string-based version technically would accept), and try to remove direct use of `LowResMeshes` below, including the `sed` line, as using two different variables for the same list looks sloppy.